src/components/LogIn/LogIn.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { View, Image, Keyboard, LayoutAnimation, Platform, UIManager, Dimensions } from 'react-native';
+import { Components } from 'expo';
+const { LinearGradient } = Components;
+
+import Input from './Input';
+import Footer from './Footer';
+import LoginBtn from './LoginBtn';
+import Header from './Header';
+import { styles } from './LogInStyle';
+
+export default class LogIn extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            inputPosition: 'relative',
+            inputBottom: 0,
+            screenWidth: Dimensions.get('screen').width,
+            SOPFontSize: 0
+        };
+        if (Platform.OS === 'android') {
+            UIManager.setLayoutAnimationEnabledExperimental(true);
+        }
+    }
+
+    componentWillMount() {
+        const toSetProperFontSize = 20.5;
```

Comment:
Zasto je ovde bas 20.5? Ako nema neki poseban razlog, postoji nacin da se to uradi a da se ne zakucava tacna vrednost. npr:
`this.setState({SOPFontSize: this.state.screenWidth / (this.state.screenWidth * 0.055)});`